client/src/ui/deckbuilder/DeckBuilderColumn.vue

```diff
@@ -1,12 +1,13 @@
 <template>
   <div
       class="_column"
-      :class="{columnDrop: dropTargetIndex !== null}"
+      :class="{columnDrop: dropTargetIndex !== null,
+               horizontal}"
       @dragover="dragOver"
       @dragleave="dragLeave"
       @dragend="dragEnd"
       @drop="drop"
-      >
+  >
```

Comment:
Nit: indent with params (here and below)

Larger comment: I don't have a strong opinion on whether these should be open-brace-aligned (like here) attribute-aligned (like in other files), I just want us to be consistent. Do you have an argument one way or the other?

Reply:
I don't have a real argument for either one, but the IntelliJ Vue plugin doesn't give me the option to attribute-align them, so these keep sneaking in from the auto-formatter.